.travis.yml

```diff
@@ -44,6 +44,9 @@ script:
   - find src/*.rs | xargs rustfmt --write-mode=diff
   # Available until `cargo fmt` landed in stable rust 1.24
   # - cargo fmt -- --write-mode=diff;
+  - |
+    clang-format native/* -output-replacements-xml |
+    grep -c "<replacement " > /dev/null; test $? -eq 1
```

Comment:
I think it's better to actually write diff if any, it may help to identify problem for people with different `clang-format` settings.

Reply:
What do you mean by `write diff`?

Using `clang-format` locally to identify problems with different settings is easier than on Travis...

Reply:
I can understand where you are coming from and see how this is useful. `clang-format` has a reasonable good reputation among C/C++ world so I would believe local and CI results would be the same.

P.S. `clang-format` output xml is hard to parse, say, the following example:

```
<replacements xml:space='preserve' incomplete_format='false'>
<replacement offset='16849' length='1'>&#10;</replacement>
</replacements>
```

If we really wanted to show the difference, we might need `clang-format-diff`. I would prefer to add that if the clang format check on CI becomes a problem and people need to debug it repeatedly.

Reply:
Yes, I see your point.

> If we really wanted to show the difference, we might need clang-format-diff. I would prefer to add that if the clang format check on CI becomes a problem and people need to debug it repeatedly.

Ok, but it seems that adding it is as simple as `diff -u <(cat src/*) <(clang-format src/*)`

Reply:
Nice tip! In this way, I also don't need to do the weird grep/test afterwards!